rbc/omnisci_backend/omnisci_bytes.py

```diff
@@ -33,14 +40,50 @@ def match(self, other):
         if other.is_string:
             return 2
 
-
-BytesPointer = BufferPointer
+    def tonumba(self, bool_is_int8=None):
+        ptr_t = typesystem.Type(self.element_type, '*', name='ptr')
+        size_t = typesystem.Type.fromstring('size_t sz')
+        extra_members = tuple(map(typesystem.Type.fromobject, self.buffer_extra_members))
+        buffer_type = typesystem.Type(
+            ptr_t,
+            size_t,
+            *extra_members
+        )
+        buffer_type._params['NumbaType'] = BufferType
+        buffer_type._params['NumbaPointerType'] = BytesPointer
+        numba_type = buffer_type.tonumba(bool_is_int8=True)
+        if self.pass_by_value:
+            return numba_type
+        return BytesPointer(numba_type)
```

Comment:
```suggestion
```

Instead of copying the whole method, I'd suggest extending `OmnisciBufferType.tonumba` as follows:
```python
class OmnisciBufferType:
    @property
    def numba_pointer_type(self):
        return BufferPointer
    ...
    def tonumba(self):
        ...
       buffer_type._params['NumbaPointerType'] = self.numba_pointer_type
       ...

class OmnisciBytesType(OmnisciBufferType):
    ...
    @property
    def numba_pointer_type(self):
        return BytesPointer
    ...
```